### DISCUSSION TITLE
How do I set download speed limits?

### DISCUSSION THREAD

**Discussion by kimtree** [transmission/transmission#2516] [Q&A]
**Environment**
Mac OS 10.14.5(18F132)
Transmissionbt 2.94 (d8e60ee44f)

**Problem**
I want to set global download bandwidth 450000KB/s which is 450Mbps, but it didn't work. It's important for me since I'm using 500Mbps internet and I need to set QoS for torrent usage.

**Comment by Pentaphon** [comment]
Your number seems to be correct...
